Content.Server/Objectives/Conditions/OtherTraitorAliveCondition.cs

```diff
@@ -0,0 +1,58 @@
+using Content.Server.Objectives.Interfaces;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Utility;
+
+namespace Content.Server.Objectives.Conditions
+{
+    public abstract class OtherTraitorAliveCondition : IObjectiveCondition
+    {
+        protected Mind.Mind? Target;
+        public abstract IObjectiveCondition GetAssigned(Mind.Mind mind);
+
+        public string Title
+        {
+            get
+            {
+                var targetName = string.Empty;
+
+                if (Target == null)
+                    return Loc.GetString("objective-condition-other-traitor-alive-title", ("targetName", targetName));
+
+                if (Target.CharacterName != null)
+                    targetName = Target.CharacterName;
+                else if (Target.OwnedEntity is {Valid: true} owned)
+                    targetName = IoCManager.Resolve<IEntityManager>().GetComponent<MetaDataComponent>(owned).EntityName;
+
+                return Loc.GetString("objective-condition-other-traitor-alive-title", ("targetName", targetName));
+            }
+        }
+
+        public string Description => Loc.GetString("objective-condition-other-traitor-alive-description");
+
+        public SpriteSpecifier Icon => new SpriteSpecifier.Rsi(new ResourcePath("Objects/Misc/bureaucracy.rsi"), "folder_red");
+
+        public float Progress => (!Target?.CharacterDeadIC ?? true) ? 1f : 0f;
+
+        public float Difficulty => 1.75f;
+
+        public bool Equals(IObjectiveCondition? other)
+        {
+            return other is OtherTraitorAliveCondition kpc && Equals(Target, kpc.Target);
+        }
+
+        public override bool Equals(object? obj)
+        {
+            if (ReferenceEquals(null, obj)) return false;
+            if (ReferenceEquals(this, obj)) return true;
+            if (obj.GetType() != GetType()) return false;
+            return Equals((OtherTraitorAliveCondition) obj);
+        }
+
+        public override int GetHashCode()
+        {
+            return Target?.GetHashCode() ?? 0;
+        }
```

Comment:
Is this needed?

Reply:
All the other objectives have it and it's part of the objective class so maybe?

Reply:
It could be reused for objectives to keep a non-traitor alive, I'll change the naming a bit